client/README.md

```diff
@@ -6,6 +6,14 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm install`
+
+Installs npm dependecies.
+If using a Windows or Linux machine, run `npm install --no-optional`.
+If using Mac OS, run `npm install`.
```

Comment:
Should we be using `npm ci` instead of `npm install` so that it will respect the package-lock.json and throw an error if package-lock.json and package.json are out of sync?

Reply:
Yes I think that is better. It seems with `npm ci`, I also do not get the severity vulnerabilities. Should I take out the note about that?